systemd/afterburn-checkin.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Descitpion=Afterburn (Check In)
+ConditionKernelCommandLine=|ignition.platform.id=azure
```

Comment:
Also `ConditionKernelCommandLine=|ignition.platform.id=packet`, right?

Reply:
@jlebon Packet only needs a checkin on first boot.  We could have a separate `afterburn-checkin-firstboot.service` with:

```
ConditionFirstBoot=true
ConditionKernelCommandLine=|ignition.platform.id=packet
```

I'd probably go that route, but there's also an argument that the separate unit isn't worth the trouble.

Reply:
Added